content/blog/tyranny-of-nits/index.md

```diff
@@ -37,50 +48,91 @@ It's really tempting, and really easy, to just blame the reviewer here.
+and extremely good at saying no to controversial things, because a conflict-driven stalemate or a timeout due to excessive revisions is still a no.
+
+But that's a *problem*: you have to be able to say yes to controversial decisions too!
+Axiomatically: to make a decision, you have to both be able to say no, and to say yes.
+If your project works like this, your project **can't make decisions about the things that matter.**
+You have to keep making decisions, day after day, or your momentum will die and everything will slowly rot.
+It doesn't even matter if they're "right": the very fact that a decision was made staves off this decay.
+
+The classical solution here is to have a BDFL, or perhaps a code owner, who can shut down discussion, merge PRs over objections and close issues as "not planned".
+As long as they're not overworked (never happens), things function pretty smoothly.
+
+But that's not the only patch: delegation, binding votes or [more sophisticated consensus-based decision making processes](https://www.sociocracyforall.org/sociocracy/) all offer another way.
+Debate these and experiment: I think any of them can work just fine.
+Just don't ignore this problem!
 
-## Automation for humans
+## Nit-pick arithme-tick
 
-Axiomatically, there are two ways to solve nitpicking in code review.
-First, reduce the amount of nits that need picking.
-Ensure that code that comes in is free of obvious trivial mistakes.
+As you might guess, there are two ways to solve nitpicking in code review: reduces the nits that need to be picked, and reduce the rate at which pickable nits are picked at.
+
+The first strategy is quite effective:
+ensure that code that comes in is free of obvious trivial mistakes.
 Linters and formatters are great for that!
 
 But just as importantly: invest in paying off your tech debt and clarifying your architecture.
 If it's *obvious* how to do something, you won't have to waste everyone's time explaining that someone did something the wrong way.
 Get those eager new contributors to help, seriously.
 
-Secondly, you can reduce the maount of nits that are picked.
-I don't actually think that telling reviewers "don't worry about this stuff" is generally a useful strategy.
+On the other hand, you could try to reduce the maount of nits that are picked.
+I don't actually think that telling reviewers "don't worry about this stuff" is generally a useful strategy: it tends to breed resentment and makes them stay quiet on helpful low-cost improvements.
 Instead, get them to focus on splitting apart their "would be nice" feedback from "this must be fixed" feedback.
 
-If you want to implement *any* of these changes effectively you need automation: whether that's CI for the machines or written guidelines for humans.
+By the way, if you want to implement *any* of these changes effectively you need automation: whether that's CI for the machines or written guidelines for humans.
```

Comment:
This is a nice aside, but I think it breaks the flow a bit. Maybe switch to a footnote instead?